From: Ilya Dryomov <idryomov@gmail.com>
Date: Wed, 27 Jun 2018 16:38:13 +0200
Subject: libceph: amend "bad option arg" error message
Git-commit: 2f56b6bae73b2d65ef4816ca89341facc53d3361
Patch-mainline: v4.19-rc1
References: FATE#324714

Don't mention "mount" -- in the rbd case it is "mapping".

Signed-off-by: Ilya Dryomov <idryomov@gmail.com>
Acked-by: Luis Henriques <lhenriques@suse.com>
---
 drivers/block/rbd.c    |    2 +-
 fs/ceph/super.c        |    3 +--
 net/ceph/ceph_common.c |    3 +--
 3 files changed, 3 insertions(+), 5 deletions(-)

--- a/drivers/block/rbd.c
+++ b/drivers/block/rbd.c
@@ -804,7 +804,7 @@ static int parse_rbd_opts_token(char *c,
 	if (token < Opt_last_int) {
 		ret = match_int(&argstr[0], &intval);
 		if (ret < 0) {
-			pr_err("bad mount option arg (not int) at '%s'\n", c);
+			pr_err("bad option arg (not int) at '%s'\n", c);
 			return ret;
 		}
 		dout("got int token %d val %d\n", token, intval);
--- a/fs/ceph/super.c
+++ b/fs/ceph/super.c
@@ -219,8 +219,7 @@ static int parse_fsopt_token(char *c, vo
 	if (token < Opt_last_int) {
 		ret = match_int(&argstr[0], &intval);
 		if (ret < 0) {
-			pr_err("bad mount option arg (not int) "
-			       "at '%s'\n", c);
+			pr_err("bad option arg (not int) at '%s'\n", c);
 			return ret;
 		}
 		dout("got int token %d val %d\n", token, intval);
--- a/net/ceph/ceph_common.c
+++ b/net/ceph/ceph_common.c
@@ -396,8 +396,7 @@ ceph_parse_options(char *options, const
 		if (token < Opt_last_int) {
 			err = match_int(&argstr[0], &intval);
 			if (err < 0) {
-				pr_err("bad mount option arg (not int) "
-				       "at '%s'\n", c);
+				pr_err("bad option arg (not int) at '%s'\n", c);
 				goto out;
 			}
 			dout("got int token %d val %d\n", token, intval);
